web-marketplace/src/components/organisms/PostForm/PostForm.tsx

```diff
@@ -250,10 +250,11 @@ export const PostForm = ({
         description={
           <Body>
             <Trans>
-              '5MB max. Supported file types include text, spreadsheets, images,
-              and video files. '
+              5MB max. Supported file types include text, spreadsheets, images,
+              and video files.
             </Trans>
-            <Link href="#">
+            &nbsp;
```

Comment:
I think it'd be better to use `padding` instead of `&nbsp;`

Reply:
Why? Using a white-space means it will adjust based on the font size so we would need to provide two padding values for mobile/desktop to get the same result.
I thought this made sense since the two elements are displayed inline, just like one single sentence but maybe there are other reasons I'm not aware of?

Reply:
As `&nbsp;` adjusts with font size we can end up losing control over the space between the elements if for example the browser settings override our font sizes. Also, using CSS for spacing keeps content and style separate, which is a good practice. I think using margin or padding is a more reliable way to control spacing between inline elements and it's easier to update and maintain if the design changes.

Reply:
I agree but I believe in this particular case, it's rather a matter of content than spacing. We really want this to fill like it should be just a paragraph of text. Just like we do for links within post comment for instance.

<img width="435" alt="image" src="https://github.com/user-attachments/assets/2d3b7f9d-4c53-49f3-b7a5-1217c9ae214c">